src/test/groovy/net/rubygrapefruit/platform/file/WindowsFileEventsTest.groovy

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2012 Adam Murdoch
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package net.rubygrapefruit.platform.file
+
+import net.rubygrapefruit.platform.Native
+import net.rubygrapefruit.platform.internal.Platform
+import net.rubygrapefruit.platform.internal.jni.WindowsFileEventFunctions
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.IgnoreIf
+import spock.lang.Specification
+import spock.lang.Timeout
+
+@IgnoreIf({ !Platform.current().windows })
+@Timeout(20)
+class WindowsFileEventsTest extends Specification {
```

Comment:
In a follow-up PR, consider having a parametrized basic test for mac&win. For any OS-specific tests, we can create separate test files.

Reply:
Yes, that's something I actually want to do in this PR.

Reply:
Done.